src/ScottPlot/Control/Backend.cs

```diff
@@ -97,6 +97,11 @@ public class ControlBackEnd
         /// </summary>
         public event EventHandler PlottableDropped = delegate { };
 
+        /// <summary>
+        /// This event is invoked every time the plot is reset and is ready to be configured.
+        /// </summary>
+        public event EventHandler PlotInitialized = delegate { };
```

Comment:
I haven't read #1171 yet, but just reading the handler name and XML summary I'm still unclear what "ready to be configured" means or what the use case is. 

After reading #1171 I have a better idea. Maybe this should be renamed to `AxesInitialized`? And/or adding more info to the XML summary may be helpful...

Reply:
Aye, I think this can be better named, but it is vague because I think I need to invoke this event later, as it doesn't look like this resolves the issue.